frontend/ui/widget/container/framecontainer.lua

```diff
@@ -41,8 +41,16 @@ local FrameContainer = WidgetContainer:new{
     invert = false,
     allow_mirroring = true,
     _mirroredUI = BD.mirroredUILayout(),
+    focusable = false,
+    focus_border_size = Size.border.window * 2,
+    focus_border_color = Blitbuffer.COLOR_BLACK,
 }
 
+function FrameContainer:init()
+    self.origin_bordersize = nil
+    self.origin_border_color = nil
+end
```

Comment:
I'm always confused when I add a :init() whether the superclass :init() will be called.
WidgetContainer:init() do some stuff with dimen, so see (with some print() here and there) if it gets called of if you need to add here `WidgetContainer.init(self)`.
(In case other instances of FrameContainer needs this upper :init() stuff).